tests/ProcessManager/ChromeManagerTest.php

```diff
@@ -68,4 +68,22 @@ public function testMultipleInstances()
         $driver1->quit();
         $driver2->quit();
     }
+
+    public function testExtensionsAndExperimentalOptions()
```

Comment:
This test is examining the behaviour of two methods. Might be best to split the test into two methods where each test examines the behaviour of just one method.

- `testSetExtensions()`
- `testSetExperimentalOptions()`